spec/models/location_spec.rb

```diff
@@ -11,28 +11,54 @@
   it { is_expected.to validate_presence_of(:location_type) }
   it { is_expected.to define_enum_for(:location_type).backed_by_column_of_type(:string) }
 
+  context 'when location is a court' do
+    subject(:location) { build :location, :court }
+
+    it { expect(location.prison?).to be false }
```

Comment:
although this seems to have been an existing pattern in our test, I'm wondering if we should be testing enum types? maybe testing detained/not_detained is enough?

Reply:
Yes, tend to agree. Those were tests we had with the old implementation prior to adding the enum, so those were custom methods. They can be stripped out now.